submit-api/src/SubmitAPI/Config.hs

```diff
@@ -0,0 +1,14 @@
+module SubmitAPI.Config where
+
+data FeePolicy
+  = Strict  -- Require existing TX inputs to cover fee entirely
```

Comment:
For what purposes should we enter these structures? Transaction balancing should take place on the adapter side, why we must introduce some policies?

Reply:
That's an implementation of the TX assembler, there is no other place TX balancing can happen.